tests/Command/ImportReferentBioPictureCommandTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\DataFixtures\ORM\LoadReferentData;
+use AppBundle\Entity\Media;
+use AppBundle\Entity\Referent;
+use Tests\AppBundle\SqliteWebTestCase;
+use Tests\AppBundle\TestHelperTrait;
+
+/**
+ * @group functional
+ */
+class ImportReferentBioPictureCommandTest extends SqliteWebTestCase
+{
+    use TestHelperTrait;
+    const VALID_ARCHIVE_NAME = 'correct.zip';
+    const ARCHIVE_WITHOUT_CSV_NAME = 'archive_without_csv.zip';
+    const ARCHIVE_WITH_NOT_REFERENT_IN_DB = 'archive_with_not_referent_in_db.zip';
+    const ARCHIVE_WITH_MISSING_IMAGE_FILE = 'archive_with_missing_image_file.zip';
+    const ARCHIVES_NAME = [
+        self::VALID_ARCHIVE_NAME,
+        self::ARCHIVE_WITHOUT_CSV_NAME,
+        self::ARCHIVE_WITH_NOT_REFERENT_IN_DB,
+        self::ARCHIVE_WITH_MISSING_IMAGE_FILE,
+    ];
+
+    public function testCommandSucessImport()
+    {
+        $this->createCorrectZipArchive();
+        $output = $this->runCommand(ImportReferentBioPictureCommand::COMMAND_NAME, ['fileUrl' => self::VALID_ARCHIVE_NAME]);
+
+        $referentRepository = $this->getRepository(Referent::class);
+        $mediaRepository = $this->getRepository(Media::class);
+
+        $referent1 = $referentRepository->findOneBy(['firstName' => 'Nicolas', 'lastName' => 'Bordes']);
+        $referent2 = $referentRepository->findOneBy(['firstName' => 'Alban', 'lastName' => 'Martin']);
+
+        $this->assertContains('Import OK', $output);
+        $this->assertNotNull($mediaRepository->findOneBy(['name' => 'Nicolas Bordes']));
```

Comment:
you can use the method you created [here](https://github.com/EnMarche/en-marche.fr/pull/1662/files#diff-9c375a087cfc698b56fc9177504e2fceR10)